tests/data/dataset_readers/nlvr_test.py

```diff
@@ -16,17 +17,17 @@ def test_reader_reads(self):
                            'a', 'box', '.']
         assert [t.text for t in sentence_tokens] == expected_tokens
         agenda = [item.sequence_index for item in instance.fields["agenda"].field_list]
-        assert agenda == [56, 32, 112]
```

Comment:
Why do these keep changing?  I'd recommend making them strings that don't change, unless you really care about the particular indices here.

Reply:
In this case (and the one that's coming up), the strings would have changed too.

Reply:
Then this should definitely be a string, I think.

Reply:
Okay, done.